.idea/.gitignore

```diff
@@ -0,0 +1,197 @@
+# Default ignored files
```

Comment:
No, I just did it because I didn't know we usually create gitignore in the root directory, and found this gitignore in .idea. So I'm gonna create it in my root directory. Thanks!

Reply:
Thank you for updating at 8a39e60502ee05206a48970c73c7536125625485.

You can remove unnecessary files such as `.idea/.gitignore` and `db/.gitignore` with `git rm`.

For example
```bash
git rm .idea/.gitignore db/.gitignore
# Then please commit the change.
```